content/en/dynamic_instrumentation/symdb/python.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enable Autocomplete and Search for Python
+is_beta: true
+private: false
+code_lang: python
+type: multi-code-lang
+code_lang_weight: 10
+---
+{{< beta-callout url="#" btn_hidden="true" >}}
+Autocomplete and search are in public beta.
+{{< /beta-callout >}}
+
+## Requirements
+
+- [Dynamic Instrumentation][1] is enabled for your service.
+- Tracing library [`dd-trace-py`][6] 2.9.0 or higher is installed.
+
+## Installation
+
+Run your service with Dynamic Instrumentation enabled, and additionally enable autocomplete and search:
+
+1. Run your service with Dynamic Instrumentation enabled by setting the `DD_DYNAMIC_INSTRUMENTATION_ENABLED` environment variable to `true`.
+2. Specify `DD_SERVICE` and `DD_VERSION` [Unified Service Tags][5].
+
+Invoke your service:
+
+```shell
+export DD_SERVICE=<YOUR_SERVICE>
+export DD_ENV=<YOUR_ENV>
+export DD_VERSION=<YOUR_VERSION>
+export DD_DYNAMIC_INSTRUMENTATION_ENABLED=true
+export DD_SYMBOL_DATABASE_UPLOAD_ENABLED=true
+ddtrace-run python -m myapp
+```
```

Comment:
Is this part of step 2, or is it a new step? Either way, I assume it is part of the list of steps, right?